### DISCUSSION TITLE
Are open groups overly reliant on servers?

### DISCUSSION THREAD

**Discussion by peepo5** [oxen-io/session-desktop#2065] [General]
Are open groups being hosted via a link/server really needed? If private groups do not need hosting, why do open groups?

Thanks.

**Comment by Bilb** [comment]
Well, there is a limit of the max size of a closed group but no limit for an opengroup. The reason is because of the private keys needed to be exchanged for a closed group so messages can be decoded.

**Reply by peepo5** [reply]
Hmm. Could this limit be raised or is it a fundemental limit of the service

**Reply by Bilb** [reply]
It's been already raised from 20 to 150. 
We cannot have an unlimited amount of people on a closed group, as a limit of the way they works (a new key needs to be share to all members everytime someone is removed from the group)

**Reply by peepo5** [reply]
Sounds great. I hope in the future that the limit can be raised to maybe 300 or even 1000 (or more!) but I am guessing there are many more planned features that have priority over this. I think this would be good just because it should be easy for those less experienced with hosting things and using technology to setup groups, so that users of various experience / expertise can use the platform. Great work so far :)

**Reply by peepo5** [reply]
If it is not possible to increase the "closed" group limit, it would be cool to either make hosting a group straightforward for the average user (using lokinet) with some sort of guide / documentation. Also btw curious about this, are there any ideas around "servers"/communities being introduced later down the line? Thanks :D

**Reply by peepo5** [reply]
Random idea, but I watched some oxen interviews and I get the idea that the network needs support through oxen donations. An idea could be that people could get a subscription (not too much cost ideally) and then that oxen could be burned in the network to support it. However, the user could choose a non-hosted/private group (like discord) to boost which could increase the members count by maybe 100 members. This may be the most suitable model if something like this was possible and also only if required. Boosts would likely have to be permanent, because otherwise you would have to kick random members.
